src/component/parser/json/converter/DiagramConverter.ts

```diff
@@ -140,20 +148,28 @@ export default class DiagramConverter extends AbstractConverter<BpmnModel> {
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   private deserializeLabel(bpmnLabel: any, id: string): Label {
-    const labelStyle = bpmnLabel?.labelStyle;
-    if (labelStyle) {
-      const font = this.findFont(labelStyle);
+    if (bpmnLabel) {
+      const labelStyle = bpmnLabel.labelStyle;
+      const font = this.findFont(labelStyle, id);
```

Comment:
:information_source: I would inline `labelStyle` has this constant is only used once and named with the field which already brings enough meaning

Reply:
Done